libsolidity/analysis/ViewPureChecker.cpp

```diff
@@ -66,9 +67,9 @@ class AssemblyViewPureChecker
 	void operator()(yul::FunctionCall const& _funCall)
 	{
 		if (yul::EVMDialect const* dialect = dynamic_cast<decltype(dialect)>(&m_dialect))
-			if (std::optional<yul::BuiltinHandle> builtinHandle = dialect->findBuiltin(_funCall.functionName.name.str()))
-				if (auto const& instruction = dialect->builtin(*builtinHandle).instruction)
-					checkInstruction(nativeLocationOf(_funCall), *instruction);
+			if (yul::BuiltinFunctionForEVM const* fun = yul::resolveBuiltinFunctionForEVM(_funCall.functionName, *dialect))
```

Comment:
```suggestion
			if (yul::BuiltinFunctionForEVM const* builtin = yul::resolveBuiltinFunctionForEVM(_funCall.functionName, *dialect))
```

Same in `ConstantOptimiser.cpp`.